spec/unit/libraries/conf_helpers_spec.rb

```diff
@@ -4,6 +4,65 @@
   include CernerSplunk::ConfHelpers
 
   let(:conf_path) { '/opt/splunk/etc/system/local/test.conf' }
+  let(:existing_config) do
+    {
+      'default' => {
+        'first' => 'true',
+        'second' => 'true'
+      },
+      'other' => {
+        'something' => 'here'
+      }
+    }
+  end
+
+  describe 'evaluate_config' do
+    context 'with top-level proc' do
```

Comment:
We might as well  use `let(:subject) { evaluate_config(existing_config, config) }` since we are testing this multiple times.